### PR TITLE
Listing index page

### PR DESCRIPTION
**Pull request by bryceholcomb**

### CODE REVIEW

**Comment by bryceholcomb** [comment]
I added the listing index and show page without any styling, I haven't namespaced the routes yet for the show page since I think kit was working on that

File: app/controllers/listings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class ListingsController < ApplicationController
```
Comment (Lydias303) [inline app/controllers/listings_controller.rb]: Nice! Im glad were starting the transition. All the pending tests are a pain tho.

File: spec/feature/authenticated_user_spec.rb [hunk 2]
```diff
@@ -5,53 +5,59 @@
 
   let!(:category1) { Category.create(name: "Breakfast") }
   let!(:category2) { Category.create(name: "Lunch") }
+  let!(:listing) do
```
Comment (Lydias303) [inline spec/feature/authenticated_user_spec.rb]: :sparkles:

File: spec/feature/authenticated_user_spec.rb [hunk 3]
```diff
@@ -80,14 +86,14 @@
     expect(page).to_not have_content("Bacon")
   end
 
-  xit "can view their own page" do
+  it "can view their own page" do
     allow_any_instance_of(ApplicationController).to receive(:current_user).
                                                     and_return(valid_user)
     visit user_path(valid_user)
     expect(current_path).to eq(user_path(valid_user))
   end
```
Comment (Lydias303) [inline spec/feature/authenticated_user_spec.rb]: Awesome! SO glad we are getting some of these tests working!

**Pull request merged**